src/utilities/sequence-editor/sequence-linter.ts

```diff
@@ -473,6 +494,206 @@ function getVariableInfo(
   };
 }
 
+function validateActivateLoad(
+  node: SyntaxNode[],
+  type: 'Activate' | 'Load',
+  text: string,
+  librarySequences: LibrarySequence[],
+): Diagnostic[] {
+  if (node.length === 0) {
+    return [];
+  }
+
+  const diagnostics: Diagnostic[] = [];
+
+  node.forEach(activate => {
```

Comment:
Some of these lint rules seem like they should be checked for all commands and not just library commands.

- Missing arguments
- Extra arguments
- Default missing arguments
- Type mismatches

Can we add a more generic way of checking these things or are these checks specific enough to library sequences?

Reply:
I added a more generic way to handle

```
Missing arguments
Extra arguments
Default missing arguments
```

`Type mismatch` is a little tricky as command arguments are heavily dependent on `ampcs` fswCommands and the library sequences are `VariableDeclaration` they datatypes are very different.